### PR TITLE
Add process logger to the usage of ADB

### LINKED ISSUE #87: Return useful error messages when adb process fails
I'm trying to debug a similar issue to https://github.com/Karumi/Shot/issues/79 and https://github.com/Karumi/Shot/issues/73 but this is next to impossible because no useful error message is return. Please pip STDERR to the console.

### PR DESCRIPTION
**Pull request by pedrovgs**
### :pushpin: References
* **Issue:** Closes #87 

### :tophat: What is the goal?

Improve the log traces shown to the user when using adb.

### How is it being implemented?

We've configured a simple ``ProcessLogger``  instance when executing adb commands.

### How can it be tested?

There is no need to add automated tests for these log traces.

### CODE REVIEW

File: core/src/main/scala/com/karumi/shot/android/Adb.scala [hunk 1]
```diff
@@ -11,6 +11,10 @@ object Adb {
 class Adb {
 
   private final val CR_ASCII_DECIMAL = 13
+  private val logger = ProcessLogger(
+    o => println("Shot ADB output: " + o),
```
Comment (tonilopezmr) [inline core/src/main/scala/com/karumi/shot/android/Adb.scala:15]: I think this should be more descriptive as `output`

File: core/src/main/scala/com/karumi/shot/android/Adb.scala [hunk 2]
```diff
@@ -11,6 +11,10 @@ object Adb {
...
+    e => println(Console.RED + "Shot ADB error: " + e + Console.RESET)
```
Comment (tonilopezmr) [inline core/src/main/scala/com/karumi/shot/android/Adb.scala:16]: Same as `output` I'd change it to `error`

File: core/src/main/scala/com/karumi/shot/android/Adb.scala (hunk 1 above)
Comment (pedrovgs) [reply core/src/main/scala/com/karumi/shot/android/Adb.scala:15]: I don't get what you suggest

File: core/src/main/scala/com/karumi/shot/android/Adb.scala (hunk 2 above)
Comment (pedrovgs) [reply core/src/main/scala/com/karumi/shot/android/Adb.scala:16]: I don't understand you suggestion

File: core/src/main/scala/com/karumi/shot/android/Adb.scala (hunk 1 above)
Comment (tonilopezmr) [reply core/src/main/scala/com/karumi/shot/android/Adb.scala:15]:
```suggestion
    output => println("Shot ADB output: " + o),
```

File: core/src/main/scala/com/karumi/shot/android/Adb.scala (hunk 2 above)
Comment (tonilopezmr) [reply core/src/main/scala/com/karumi/shot/android/Adb.scala:16]:
```suggestion
    error => println(Console.RED + "Shot ADB error: " + error + Console.RESET)
```

File: core/src/main/scala/com/karumi/shot/android/Adb.scala (hunk 1 above)
Comment (pedrovgs) [reply core/src/main/scala/com/karumi/shot/android/Adb.scala]: Fixed!

File: core/src/main/scala/com/karumi/shot/android/Adb.scala (hunk 2 above)
Comment (pedrovgs) [reply core/src/main/scala/com/karumi/shot/android/Adb.scala]: Fixed!

**Pull request merged**